reference/api/documents.mdx

```diff
@@ -47,25 +129,34 @@ Documents are ordered by Meilisearch depending on the hash of their id.
 
 ```json
 {
-  "results": [
+  "results":[
     {
-      "id": 25684,
-      "release_date": "1993-01-01",
-      "poster": "https://image.tmdb.org/t/p/w1280/iuAQVI4mvjI83wnirpD8GVNRVuY.jpg",
-      "title": "American Ninja 5",
-      "overview": "When a scientists daughter is kidnapped, American Ninja, attempts to find her, but this time he teams up with a youngster he has trained in the ways of the ninja."
+      "id":364,
+      "title":"Batman Returns",
+      "overview":"While Batman deals with a deformed man calling himself the Penguin, an employee of a corrupt businessman transforms into the Catwoman.",
+      "genres":[
+        "Action",
+        "Fantasy"
+      ],
+      "poster":"https://image.tmdb.org/t/p/w500/jKBjeXM7iBBV9UkUcOXx3m7FSHY.jpg",
+      "release_date":708912000
     },
     {
-      "id": 468219,
-      "title": "Dead in a Week (Or Your Money Back)",
-      "release_date": "2018-09-12",
-      "poster": "https://image.tmdb.org/t/p/w1280/f4ANVEuEaGy2oP5M0Y2P1dwxUNn.jpg",
-      "overview": "William has failed to kill himself so many times that he outsources his suicide to aging assassin Leslie. But with the contract signed and death assured within a week (or his money back), William suddenly discovers reasons to live... However Leslie is under pressure from his boss to make sure the contract is completed."
+      "id":13851,
+      "title":"Batman: Gotham Knight",
+      "overview":"A collection of key events mark Bruce Wayne's life as he journeys from beginner to Dark Knight.",
+      "genres":[
+        "Animation",
+        "Action",
+        "Adventure"
+      ],
+      "poster":"https://image.tmdb.org/t/p/w500/f3xUrqo7yEiU0guk2Ua3Znqiw6S.jpg",
+      "release_date":1215475200
     }
   ],
-  "offset": 0,
-  "limit": 2,
-  "total": 500134
+  "offset":0,
+  "limit":2,
+  "total":5403
 }
```

Comment:
Why are we removing the space between `:` and value? Have we decided on this recently?

Taking a quick look at other JSON blocks in other pages (geosearch, filtering, distinct attribute), it seems we  always use that space.

Reply:
Good catch! I'll fix it